graphs.py

```diff
@@ -0,0 +1,15 @@
+import random
+from collections import defaultdict
+
+def make_complete_graph(n):
+    return make_random_graph(n, 1)
+
+def make_random_graph(n, density):
+    g = defaultdict(list)
+    for i in range(n):
+        for j in range(n):
+            if i == j:
+                continue
+            if random.random() < density:
+                g[i].append(j)
```

Comment:
Should we make this function to return a undirected graph instead? I think we are only working with undirected graphs

Reply:
Oops, I think I missed the `g[j].append(i)` by accident. Thanks for catching this!

Reply:
fixed :)